contrib/pkg/deprovision/awstagdeprovision.go

```diff
@@ -2,13 +2,18 @@ package deprovision
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
+	"path/filepath"
 	"strings"
 
-	"github.com/openshift/installer/pkg/destroy/aws"
+	"github.com/fsnotify/fsnotify"
```

Comment:
Was there any consideration for using `github.com/openshift/library-go/pkg/controller/fileobserver` instead?

Reply:
No was not aware of that, any benefits? The type is uncommented. I'm going to assume yes and replace it.